src/Analyzers/OM/ParsedItemsContext.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.BackEnd.Logging;
+using Microsoft.Build.Construction;
+
+namespace Microsoft.Build.Experimental;
+public enum ItemType
```

Comment:
This is an SDK thing. I understand the convenience it's adding but it I would not expose it from here.